src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Formatters/EnumAsStringFormatter`1.cs

```diff
@@ -52,4 +52,67 @@ public T Deserialize(ref MessagePackReader reader, MessagePackSerializerOptions
             return value;
         }
     }
+
+    public sealed class GenericEnumFormatter<T> : IMessagePackFormatter<T>
```

Comment:
IMO this class should be in its own file. Having it in `EnumAsStringFormatter`1.cs` isn't very discoverable.

Reply:
Yes.

Actually, there is still a lot of work to deal with Unity.
As a starting point, once this PR was merged, I was thinking of organizing it.
(There are also annoying problems like Directory.Build.props)

Reply:
Yes, speaking of directory.build.props, the only fix I can imagine for that will lead to most of the files being moved. So we should coordinate so that we're not both doing major reorganizing of files at the same time.